src/containers/ErrorMessages.js

```diff
@@ -24,10 +25,11 @@ function errorMessage(error) {
 
 function mobileClientError(error) {
   try {
+    const appName = get(error, 'response.data.details.name');
```

Comment:
I would move this line inside the switch case as it is not needed until then.

Reply:
If this is move inside the switch case, there is errors raised by `eslint`. Should I still move it.

Reply:
Once you add curly brackets to your switch case, these warnings will go away.

Reply:
:+1: